### ISSUE TITLE
Proposal: Swap the order of section titles and icons

### ISSUE THREAD

**Issue by sschuberth** [jmongard/Git.SemVersioning.Gradle#63]
Just a proposal: What do you think about swapping the order of section titles and icons, so that icons come first and look a bit like funny bullet points.

So e.g.

Bug Fixes 🐞

would become

🐞 Bug Fixes

Or is there some kind of sorting based on the section titles somewhere that would break by this?

**Comment by jmongard** [comment]
Yes, there is sorting involved, alphabetically. But the order is not that important I think. Fixes and features are always sorted first because they are added to a list of types to show first. (This list can be used to order every type as you want.)

Right now there are two sets of standard texts. "Default" with emojis last and plain text.
You can also redefine all text as you please. 

I could easily add a third set of standard texts with emojis placed first if you think that will look better?
Have you looked at any other change log with emojis placed first?

**Comment by sschuberth** [comment]
> Have you looked at any other change log with emojis placed first?

Not really, it's just something that pops to my eyes every time I look at our [own release notes](https://github.com/oss-review-toolkit/ort/releases). Not having these icons vertically aligned somehow looks "noisy" to me.

**Comment by jmongard** [comment]
Here is an example and it looks slightly better:
https://github.com/riok/mapperly/releases

I will have to think how to best implement it.

**Comment by sschuberth** [comment]
> Here is an example and it looks slightly better: https://github.com/riok/mapperly/releases

For reference, that's using [this configuration](https://github.com/riok/mapperly/blob/ad4b28e177cefaabeb7ccc694dd65baee3b9cd65/.github/release-drafter.yml#L10-L19) for [Release Drafter](https://github.com/release-drafter/release-drafter).

**Comment by jmongard** [comment]
The order is now swapped in version 0.13.0
